backend/api/whereaboutsApi.ts

```diff
@@ -76,20 +76,12 @@ export = class WhereaboutsApi {
     return this.get(context, `/video-link-rooms/${agencyId}`)
   }
 
-  public getVideoLinkBookings(
+  public findVideoLinkBookings(
```

Comment:
renamed this with a prefix of find as it is now a post so naming `getVideoLinkBookings` seemed strange.